### ISSUE TITLE
Generating path objects from a list of node objects

### ISSUE THREAD

**Issue by VincenzoPerri** [pathpy/pathpy#52]
it is currently not possible to do the following.

`path_object = pp.Path([node for node in node_objects])`

**Comment by IngoScholtes** [comment]
I agree that this would be a useful feature, especially when using paths to represent random walk sequences. However, this would make it impossible to check whether a path is valid in the sense of being a list of consecutive edges
